namespaces/cloud-platform-live-0.k8s.integration.dsd.io/categorisation-tool-dev/resources/main.tf

```diff
@@ -0,0 +1,27 @@
+terraform {
+  backend "s3" {}
+}
+
+provider "aws" {
+  region = "eu-west-1"
+}
+
+module "ecr-repo" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=1.0"
```

Comment:
ref=2.0 pls, new release https://github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials/releases

Reply:
thanks - I'll update it.  Maybe the docs should be changed - https://ministryofjustice.github.io/cloud-platform-user-docs/01-getting-started/003-ecr-setup/#creating-the-registry